kod/object/active/holder/room/monsroom/desertafterlife/desertALdunes.kod

```diff
@@ -0,0 +1,64 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+DesertAfterlifeDunes is DesertAfterlife
+
+% This room is free of monsters.
+% Players may walk east (brighter) toward the Prism of Life.
+% They may also walk west (darker) toward the Prism of Death.
+% The souls of centuries of dead nomads who could not afford a deal with
+%  Xaerdun wander here. They can be found all over the other afterlife rooms.
+
+constants:
+
+   include blakston.khd
+
+resources:
+
+   room_desertALdunes = desertaldunes.roo
```

Comment:
The actual .roo file for this room has capital letters but is listed here without. Please use all lowercase for all .roo files and any reference to them, as any difference here will cause the rooms to not load on linux blakserv.